web/packages/plugins/dmt-app/src/components/context-menu/ContextMenu.tsx

```diff
@@ -163,6 +163,7 @@ export const NodeRightClickMenu = (props: {
   children: any
 }) => {
   const { node, children, removeNode } = props
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore